src/fmu/dataio/_model/fields.py

```diff
@@ -298,6 +298,9 @@ class Realization(BaseModel):
     """The universally unique identifier for this realization. It is a hash of
     ``fmu.case.uuid`` and ``fmu.iteration.uuid`` and ``fmu.realization.id``."""
 
+    is_reference: bool = Field(default=False)
```

Comment:
Will this default it to `False` for _all data objects_ produced? If that's the case, that was not the intention. It should only be used within the `Realization` object, i.e objects of class="realization". So not the realization _block_ of every data object. We may have confused each other here 😕. Or am I confusing myself right now?

@equinor-ruaj do you want to add to the confusion?

Reply:
You are completely correct @perolavsvendsen .. Nice catch 👍  this is on me, I understood the intention but messed up the implementation! In my head the new Realization object did not share pydantic models with the data models.. but they do of course! Will make necessary changes 😄 

..the schema is starting to get complicated 🤔